client/sagas/execute-challenge-saga.js

```diff
@@ -99,29 +99,34 @@ export default function executeChallengeSaga(action$, getState) {
       const { files } = state.challengesApp;
```

Comment:
If I'm honest its the first place I tried. The transformers didn't even come into my thought process. I will refactor.